awx/main/models/projects.py

```diff
@@ -466,6 +466,15 @@ def _get_related_jobs(self):
             models.Q(ProjectUpdate___project=self)
         )
 
+    def delete(self, *args, **kwargs):
+        path_to_delete = self.get_project_path(check_if_exists=False)
+        r = super(Project, self).delete(*args, **kwargs)
+        if path_to_delete:
+            # path is None in cases of manual projects, etc.
```

Comment:
Is that true?  Seems simpler/clearer to just check `if not self.scm_type:`

Reply:
Ah! We need _both_ checks. I got confused.

Yes, we shouldn't do this for manual projects, but there are other cases where a normal project has never been updated, or some other reason that `local_path` is blank, in that case we _also_ don't want to submit this task, so we should check both right here.